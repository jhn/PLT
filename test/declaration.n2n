;Test the multiple declaration of a variable local scoping;

a: Int = 100

fn declaration () -> Void {
	print(a)
	a = 300
	print(a)
	a: Int = 1  ;Not sure what happens when declared again;
	print(a)

	b: Int = 200
	print(b)

}