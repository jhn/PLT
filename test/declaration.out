100
300
1
200
